### PR TITLE
Added label for Search form button & fixed sidebar menu improper display

### PR DESCRIPTION
**Pull request by natnael2012**
Search form button requires label for accessibility. The function or purpose of the button can not be presented to screen reader users.

### CODE REVIEW

**Comment by natnael2012** [comment]
Improper display of Sidebar menu on pages with larger height #3381

File: kalite/distributed/static/css/distributed/sidebar.css [hunk 1]
```diff
@@ -24,9 +24,9 @@ Nav Panel
 
 .sidebar-panel {
     -webkit-transform: translateZ(0);
-    background: #5AA685;
+    background: #335c49;
```
Comment (jtamiace) [inline kalite/distributed/static/css/distributed/sidebar.css]: Please change the color back to the other one. It is part of our current color scheme.

**Comment by jtamiace** [comment]
@natnael2012 can you post a screen shot of the fixed screen size as well as the new search bar that you have? Can't tell the difference between these screenshots and the ones from the referenced issue.

**Comment by natnael2012** [comment]
Hi @jtamiace regarding the search form label, I have added aria-label attribute and it does not affect the layout. But it only assist screen reader users by giving description. Here is screenshot

search_label

And for the fixed screen size, below is screenshot. please take a look at the scroll bar at the right. The first screenshot is at the top of the page and the later is when scroll down to bottom.And you can see the sidebar menu improper display. This issue happens only on pages with larger height.

sidebar1

When scroll down to bottom of the page
sidebar2

**Comment by natnael2012** [comment]
@aronasorman please assign Jessica on this pull request so I can discuss regarding the web color. Thanks

File: kalite/distributed/templates/distributed/base.html [hunk 2]
```diff
@@ -171,10 +171,10 @@
                               {# Search box for responsive nav #}
                               <div class="visible-xs">
                                   <li id="search_input">
```
Comment (jtamiace) [inline kalite/distributed/templates/distributed/base.html]: Hi @natnael2012 please accept my apology for this late reply. we would like to merge your changes, but there are a few conflicts that need to be fixed. we've removed this first search bar for cleanup. can you please pull in the latest develop branch and make this change?

**Comment by jtamiace** [comment]
Also, I'm sorry for missing this conversation about color! What did you have in mind?

File: kalite/distributed/templates/distributed/base.html (hunk 2 above)
Comment (natnael2012) [inline kalite/distributed/templates/distributed/base.html]: @jtamiace its okay for the late reply, I will pull the latest develop branch and will update.

**Comment by natnael2012** [comment]
@jtamiace Regarding the color issue, as long as accessibility is concerned, the text elements are hardly readable by users with low vision. The menu Items has Contrast Ratio: 2.91:1, (background color #5aa685 and foreground color #fff). This is very low contrast between foreground and background colors. I have also opened an issue https://github.com/learningequality/ka-lite/issues/3297

**Comment by jtamiace** [comment]
@natnael2012 That's a good point. I will bring this up with the design team that we have here and discuss the issue that you opened! For now though, I will merge the changes that you have made. We will be in touch regarding the color issue.

**Comment by jtamiace** [comment]
I should have paid closer attention before I merged this commit, but @natnael2012 looks like the changes that you made to base.html were overridden by your fixes for the merge conflicts. Could you please open a new pull request with the changes you have made?

**Pull request merged**